### DISCUSSION TITLE
SQLP examples with stdin

### DISCUSSION THREAD

**Discussion by slonyik** [dathere/qsv#1080] [Q&A] [answered]
Could you please add example of polars/sqlp  using the stdin, please ?

`cat file.cvs | qsv  sqlp - 'select * from stdin' `

it took me a while to find out the syntax
having it in the sqlp help might spare time for others.

Thanks

**Comment by ondohotola** [comment, marked as answer]
RTFM :-)-O see issue #1026 and you can do something like

```
cat file.cvs | qsv rename _all_generic | qsv  sqlp -Q - 'select _col_1 as A, _col_2 as B from stdin' 
```

and _pipe_ (take) it from there

**Reply by slonyik** [reply]
Thanks @ondohotola 👍 

The ask here is to add that example to the help* text of sqlp.

Truth is, I did not read/search the discussions just  code/repo search - my fault.
* help i.e.  the M in RTFM ;-)

**Comment by jqnatividad** [comment]
And I also added an example for stdin and `read_ndjson` as well.

https://github.com/jqnatividad/qsv/commit/64c85ae627b9e7e2b1b14ff51cc18cca164f9e6a

FYI, Polars SQL is evolving quite fast and the documentation on it is still a bit sparse, that's why I linked to this tracking issue from `sqlp` usage text.

https://github.com/pola-rs/polars/issues/7227

**Reply by slonyik** [reply]
👍 
Thanks !
